package.json

```diff
@@ -34,6 +34,7 @@
     "graphql-server-express": "1.2.0",
     "graphql-subscriptions": "0.5.4",
     "graphql-tools": "2.6.1",
+    "memorystore": "^1.6.0",
```

Comment:
I like exact versions, so with out the ^.

I've been burned by unexpected updates before :)